corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -237,21 +297,24 @@ def _ledgers_per_case(self):
 
         case_types = case_types_result.aggregations.types.keys
 
-        self.stdout.write('\nCase Types with Ledgers')
+        resource_model['case_types_avg_lifespans'] = []
```

Comment:
why is this part of the 'ledgers' data - it doesn't seem related at all

Reply:
Oh, it's not. This was part of some change I made and I kinda build it into something that is not represented by "case_types_avg_lifespans" anymore. I changed it now.